index.html

```diff
@@ -2,12 +2,27 @@
 <html lang="en">
     <head>
         <meta charset="UTF-8" />
-        <link href='/favicon.ico' rel='shortcut icon' type='image/x-icon'>
-        <link href='/apple-touch-icon.png' rel='apple-touch-icon' sizes='180x180'>
-        <link href='/favicon-32x32.png' rel='icon' sizes='32x32' type='image/png'>
-        <link href='/favicon-16x16.png' rel='icon' sizes='16x16' type='image/png'>
+        <link href="/favicon.ico" rel="shortcut icon" type="image/x-icon" />
```

Comment:
```suggestion
        <link href="/favicon.ico" rel="icon" type="image/x-icon" />
```

Reply:
I don't think we use "shortcut" in `rel` for favicon anymore. It used to be a thing though.

Reply:
I didn't know that, learned something new today. I thought it was maybe some there for other reasons

Reply:
Actually I learned about this recently as well